apps/examples/logs_dynamic/logs_dynamic_main.c

```diff
@@ -0,0 +1,44 @@
+/****************************************************************************
+ *
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <tinyara/serial/tioctl.h>
+#include <fcntl.h>
+#include<sys/ioctl.h>
+
+
+/****************************************************************************
+ * hello_main
+ ****************************************************************************/
+
+#ifdef CONFIG_BUILD_KERNEL
+int main(int argc, FAR char *argv[])
+#else
+int logs_dynamic_main(int argc, char *argv[])
+#endif
+{
+	int fd = open("/dev/console", O_RDWR);
+	ioctl(fd, LOG_DYNAMIC, NULL);
```

Comment:
This would not be the correct way to use an IOCTL call I feel, please check if you do ultimately intend to keep the functionality this way.